test/e2e/testdata/wolf-rbac/start.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```

Comment:
Can we run the wolf rbac server on kubernetes?

Reply:
Yes, but it will be troublesome. The database needs to be deployed and initialized, and there are many changes to be made.

Reply:
I think we can run it with the easiest way since it's just a test dependency.

Reply:
OK, just an advice.